autobump-core/src/test/java/com/github/autobump/core/model/usecases/AutobumpUseCaseTest.java

```diff
@@ -0,0 +1,120 @@
+package com.github.autobump.core.model.usecases;
+
+import com.github.autobump.core.model.Bump;
+import com.github.autobump.core.model.CommitResult;
+import com.github.autobump.core.model.Dependency;
+import com.github.autobump.core.model.DependencyBumper;
+import com.github.autobump.core.model.DependencyResolver;
+import com.github.autobump.core.model.GitClient;
+import com.github.autobump.core.model.GitProvider;
+import com.github.autobump.core.model.PullRequest;
+import com.github.autobump.core.model.UrlHelper;
+import com.github.autobump.core.model.Version;
+import com.github.autobump.core.model.VersionRepository;
+import com.github.autobump.core.model.Workspace;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class AutobumpUseCaseTest {
+
+    private GitProvider gitProvider;
+
+    private GitClient gitClient;
+
+    private DependencyResolver dependencyResolver;
+
+    private VersionRepository versionRepository;
+
+    private DependencyBumper dependencyBumper;
+
+    private UrlHelper urlHelper;
+
+    private URI uri;
+
+    @BeforeEach
+    void setUp() throws URISyntaxException {
+        makeMocks();
+        setUpMocks();
+    }
+    @SuppressWarnings("ExecutableStatementCount")
```

Comment:
Let's create a new issue on GitHub to revise this in the next sprint.
Not necessary to fix this in this pull request scope.